### ISSUE TITLE
Failing PropTypes for FirestoreConnectWrapped component

### ISSUE THREAD

**Issue by braican** [prescottprue/react-redux-firebase#758]
It looks like in the recent update to the 3.0.0-beta, there was an adjustment to the props passed to the `FirestoreConnectWrapped` component where the `firebase` prop was updated to match the value passed to the `dispatch` prop:

https://github.com/prescottprue/react-redux-firebase/pull/756/files#diff-31f278ad6811ed873821e17ba66c026bR108

However in the `FirestoreConnectWrapped` component, the propTypes declarations weren't updated: `dispatch` expects a function where the `firebase` prop expects an object. This is causing the prop types error since the same value is being passed to the component for both props.

https://github.com/prescottprue/react-redux-firebase/blob/8f9bf491757dea675374495f95c083ee8f532579/src/firestoreConnect.js#L92

**What is the current behavior?**

Failing prop types check in the `FirestoreConnectWrapped` component:

```
Warning: Failed prop type: Invalid prop `firebase` of type `function` supplied to `FirestoreConnectWrapped()`, expected `object`.
```

**What is the expected behavior?**

No prop types warnings or errors.

**Which versions of dependencies, and which browser and OS are affected by this issue? Did this work in previous versions or setups?**

react-redux-firebase: 3.0.0-beta

**Comment by prescottprue** [comment]
Whoops, thanks for reporting!

**Comment by prescottprue** [comment]
Fixed in [v3.0.0-beta.2 release](https://github.com/prescottprue/react-redux-firebase/releases/tag/v3.0.0-beta.2). Reach out if anything else isn't working as expected!
